### PR TITLE
Create derisking-guide-update.md

### PR DESCRIPTION
**Pull request by ameliaswong**
# Pull request summary

(Optional) Closes #(issue_number).

## Reminder - please do the following before assigning reviewer

- [ ] Update readme
- [ ] For frontend changes, ensure design review
- [ ] For content changes beyond typos, add Ron Bronson as a reviewer

And make sure that automated checks are ok

- fix houndci feedback
- ensure tests pass
- federalist builds
- no new SNYK vulnerabilities are introduced

### CODE REVIEW

**Review by quarterback — APPROVED** [review]

File: content/posts/derisking-guide-update.md
Comment (beechnut) [inline content/posts/derisking-guide-update.md:1]: The post filename must follow the conventions set by the rest of the posts, YYYY-MM-DD-post-slug.md

Comment (beechnut) [inline content/posts/derisking-guide-update.md:1]: There's [no linked blog-drafts issue](https://github.com/18F/blog-drafts/issues). Between that and the evidence that this hasn't had a plain language review, it looks like this post is skipping the standard approval process.

Is that what's supposed to happen here? @quarterback

File: content/posts/derisking-guide-update.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ excerpt: |
 image: 
 ---
 
-Today we launch [De-risking Government Technology Guide 2.0](https://guides.18f.gov/derisking-government-tech/), the first update of the 18F De-risking Guide since it was published in 2020. Find all the recommendations from the original guide’s two parts—the State Software Budgeting Handbook and the Federal Field Guide—now in one unified package. Plus, a new, in-depth section on vendor management! 
+Today we launch a new version of the [De-risking Government Technology Guide](https://guides.18f.gov/derisking-government-tech/), its first update since being published in 2020. Find all the recommendations from the original guide’s two parts—the State Software Budgeting Handbook and the Federal Field Guide—now in one unified package. Plus, a new, in-depth section on vendor management!
```
Comment (beechnut) [inline content/posts/derisking-guide-update.md:15]: concern: There's evidence that this hasn't gone through plain-language review: it uses some passive voice, there are sentence fragments, etc. Is there a plan to do a plain language review or check this against the 18F style guide?

File: content/posts/derisking-guide-update.md [hunk 2]
```diff
@@ -29,5 +29,5 @@ These changes respond to current government agency needs. As part of the Office
 
 The changes also reflect the core 18F value to iterate. The guide's original authors hoped it would be revised over time. As we're able, we'll continue to do so.
 
-Check out the [new guide online](https://guides.18f.gov/derisking-government-tech/) and in [PDF](/). Share it around. Suggestions, questions, and feedback are welcome! Please reach out through the 18F Guides GitHub repo.
+Check out the [new guide online](https://guides.18f.gov/derisking-government-tech/) and in [PDF](/). Share it around. Suggestions, questions, and feedback are welcome! Please reach out through the [18F Guides GitHub repo](https://github.com/18F/guides/issues/new/choose).
```
Comment (beechnut) [inline content/posts/derisking-guide-update.md:32]: change: There is a hidden broken link here: the PDF link just links to the site's root path (`/`). This wouldn't be caught by our link checkers, and should be updated so this doesn't fall through the cracks.

**Review by beechnut — CHANGES_REQUESTED** [review]

File: content/posts/derisking-guide-update.md
Comment (ameliaswong) [reply content/posts/derisking-guide-update.md:1]: There is a blog-drafts issue, but I didn't know it was supposed to be linked: https://github.com/18F/blog-drafts/issues/900

File: content/posts/derisking-guide-update.md (hunk 1 above)
Comment (ameliaswong) [reply content/posts/derisking-guide-update.md:15]: It did go through a plain-language review, but some things always slip through the cracks.

**Comment by ameliaswong** [comment]
@beechnut I think I've resolved all the requested changes. If you're happy with them, do you merge or do I? (It's been a year since I did my last post, so I've forgotten the process.)

**Review by ameliaswong — COMMENTED** [review]
I've addressed all the requested changes, I believe.

**Review by nateborr — APPROVED** [review]
Confirmed that the updates address @beechnut's feedback and the post and links look good overall.

**Pull request merged**